core/iwasm/libraries/lib-wasi-threads/test/skip.json

```diff
@@ -1,5 +1,6 @@
 {
     "lib-wasi-threads tests": {
-        "spawn_stress_test": "Stress tests are incompatible with the other part and executed differently"
+        "spawn_stress_test": "Stress tests are incompatible with the other part and executed differently",
+        "stress_test_threads_creation": "Stress tests are incompatible with the other part and executed differently"
```

Comment:
more tests we add here more convinced I am this should all be ran as part of the test runner. What would it take to run those tests just like we run other thread tests?

Reply:
for AOT - nothing 

for non-aot I don't even know how hard will it be (because aot is executed differently)

Reply:
Maybe we can go the opposite way and scope threads that depend on max_threads=4 (there should be only one/a couple of them)

Reply:
let's do this in another PR